packages/core/package.json

```
@@ -16,7 +16,6 @@
     "test": "test"
   },
   "dependencies": {
```

But someone will get confused again in a year or two :) It's too bad package.json comments are terrible, but this might work

```suggestion
  "dependencies": {
     "//": "@types/cls-hooked exposed in API so can't be in devDependencies",
```

---

I guess you can't nest comments in the `dependencies` block according to these comments https://stackoverflow.com/a/14221781/8185365. I'll add the comment above depdencies